### PR TITLE
Fix watch expression tree not expanding

### PR DESCRIPTION
**Pull request by wldcordeiro**
I hadn't seen that @arthur801031 had claimed the issue but talked to him on Slack about opening the PR.

Associated Issue: #1970

### Summary of Changes

* remove flex box and change width to account for close button

### Test Plan

- [x] Add a watch expression and see that the tree actually expands vertically

#### Before

screenshot_20170208_225031

#### After

screenshot_20170208_225014

### CODE REVIEW

File: src/components/SecondaryPanes/Expressions.css [hunk 1]
```diff
@@ -43,8 +43,9 @@
 }
 
 .expression-container > .tree {
-  overflow: hidden;
+  overflow: auto;
```
Comment (arthur801031) [inline src/components/SecondaryPanes/Expressions.css]: Hi @wldcordeiro 
`overflow: auto` seems to cause the scroll bar to appear when you hove over an element. Reverting overflow to `hidden` would fix this issue.

screen shot 2017-02-09 at 1 54 14 pm

Comment (jasonLaster) [inline src/components/SecondaryPanes/Expressions.css]: Thanks @arthur801031 I just tried replacing `overflow: auto` with `hidden` and it worked well.

**Comment by wldcordeiro** [comment]
@arthur801031 From my investigation this comes from `ObjectInspector` and could be fixed separately by having `text-overflow` set to ellipsis for element names?

**Comment by jasonLaster** [comment]
@wldcordeiro ObjectInspector seems like a good idea. Feel free to go ahead and do it!

**Comment by wldcordeiro** [comment]
I've been researching fixing the hover issue @arthur801031 pointed out, this is because of the `CloseButton` that is added on hover. This margin/padding 0 [here](https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-sham-modules/client/shared/components/tree.js#L306) causes issues with the easiest solution as it overrides any margin we add.

**Comment by jasonLaster** [comment]
I pushed a quick fix for the separate scroll bars

screen shot 2017-02-16 at 10 55 04 pm

**Pull request merged**